mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -13,6 +13,10 @@
 	{
 		__original();
 		this.m.RF_DamageReceived = { Total = 0.0 };
+		if (::World.Assets.getCombatDifficulty() == ::Const.Difficulty.Easy && this.getType() != ::Const.EntityType.Player)
+		{
+			this.m.BaseProperties.DamageTotalMult = 0.9;
```

Comment:
This has to happen in `onInit` because that's where the BaseProperties are initialized.